coralproject/talk
# [CORL-604] Force admin local auth via feature flag

## What does this PR do?

Adds a feature flag called `FORCE_ADMIN_LOCAL_AUTH` that forces the local authentication for the admin side of Coral on and disables the ability to turn it off.

It also shows a callout explaining why this is happening in the config settings so users know why it is greyed out.

## What changes to the GraphQL/Database Schema does this PR introduce?

Adds the feature flag `FORCE_ADMIN_LOCAL_AUTH`.

## How do I test this PR?

- Enable the feature flag `FORCE_ADMIN_LOCAL_AUTH`
- Go to Admin > Config > Authentication > Login with email authentication
- See that you can no longer toggle the Coral Admin option and it is enabled
- See that you cannot turn off Local Auth either 
 
## How do we deploy this PR?

- Enable the feature flag `FORCE_ADMIN_LOCAL_AUTH` for clients that need it
